openlibrary/templates/account/create.html

```diff
@@ -43,36 +43,41 @@ <h1>$_("Sign Up")</h1>
         $if form.note:
             <div class="note">$form.note</div>
 
-        $def screenname_url(): $_('Your URL'): https://openlibrary.org/people/<span id="userUrl">$(form.username.value or _('screenname'))</span>
+        $def screenname_url(): $_('')
 
         $:field(form.email)
         $:field(form.username, suffix=str(screenname_url()))
         $:field(form.password)
         $:field(form.password2)
 
-        <br/>
-        <label>
-            $:form.ia_newsletter.render() $:form.ia_newsletter.description
-        </label>
-
-        <hr/>
-
+        <br>
         $if form.has_recaptcha:
-            <div class="formElement">
-                <div class="label smaller lighter">$_("If you have security settings or privacy blockers installed, please disable them to see the reCAPTCHA.")</div>
-                <div class="g-recaptcha" data-sitekey="$form['recaptcha'].public_key"></div>
-                <div class="input">
-                    $if form['recaptcha'].error:
-                        <span class="invalid clearfix">$_('Incorrect. Please try again.')</span>
-                </div>
+        <div class="formElement">
+            <div class="label smaller lighter">$_("Please disable any privacy blockers to verify your identity.")</div>
+            <div class="g-recaptcha" data-sitekey="$form['recaptcha'].public_key"></div>
+            <div class="input">
+                $if form['recaptcha'].error:
+                    <span class="invalid clearfix">$_('Incorrect. Please try again.')</span>
             </div>
+        </div>
 
+        <br>
+        <b>Subscriptions</b>
+        <br>
+        <label>
+            $:form.ia_newsletter.render() $:form.ia_newsletter.description
+        </label>
+        <br>
+        <label>
+            $:form.ib_newsletter.render() $:form.ib_newsletter.description
```

Comment:
```suggestion
            $:form.ia_events_notifications.render() $:form.ia_events_notifications.description
```
Use updated name here.